src/sentry/static/sentry/app/views/settings/incidentRules/details.tsx

```diff
@@ -30,9 +30,9 @@ class IncidentRulesDetails extends AsyncView<Props, State> {
   }
 
   getEndpoints(): [string, string][] {
-    const {orgId, projectId, ruleId} = this.props.params;
+    const {orgId, ruleId} = this.props.params;
```

Comment:
can clean up projectId from params

Reply:
hmmm seems like typescript complains when i remove projectId from the `RouteParams` because these props are forwarded to the `<RuleForm>` component which requires `projectId`. I feel like that shouldn't be an issue because those are router params, but not sure how to get around it :/